src/components/vue-form-builder.vue

```diff
@@ -648,9 +648,12 @@ export default {
       const excludedLabels = [""];
 
       const filtered = this.controls.filter((control) => {
-        return control.label
-          .toLowerCase()
-          .includes(this.filterQuery.toLowerCase());
+        let filter = this.filterQuery.toLowerCase();
```

Comment:
Extract the filter query outside the `filter`  array method
```suggestion
      const filter = this.filterQuery.toLowerCase();
      const filtered = this.controls.filter((control) => {
```